hadoop-project/pom.xml

```diff
@@ -244,7 +244,7 @@
     <openssl-wildfly.version>2.1.4.Final</openssl-wildfly.version>
     <jsonschema2pojo.version>1.0.2</jsonschema2pojo.version>
     <woodstox.version>5.4.0</woodstox.version>
-    <nimbus-jose-jwt.version>9.37.2</nimbus-jose-jwt.version>
+    <nimbus-jose-jwt.version>9.37.4</nimbus-jose-jwt.version>
```

Comment:
why don't we move to 9.48?
https://mvnrepository.com/artifact/com.nimbusds/nimbus-jose-jwt/9.48